src/04-modality-specific-files/11-functional-near-infrared-spectroscopy.md

```diff
@@ -58,33 +58,35 @@ the `*_nirs.json` file.
 Additionally, in certain situations, reserved keywords MUST be used,
 When a custom made cap is used, the reserved keyword `custom` MUST be used in the
 `CapManufacturer` field.
-Simillarly, when a custom cap is used or custom modifications are made to a cap,
+Similarly, when a custom cap is used or custom modifications are made to a cap,
 then the reserved keyword `custom` MUST be used for the `CapManufacturersModelName` field.
 If no cap is used the reserved keyword `none` MUST be used in the `CapManufacturer`
 and `CapManufacturersModelName` field.
+The use of 'NIRSPlacementScheme' is RECOMMENDED, certainly when no cap or a costomized cap is used,
+and describes the positioning of the optodes.
 To clarify the usage and interaction of these fields, the following examples are provided:
 
 -   If a commercial cap such as EasyCap actiCAP 64 Ch Standard-2 was used:
     ```JSON
     "CapManufacturer": "EasyCap",
     "CapManufacturersModelName": "actiCAP 64 Ch Standard-2",
-    "NIRSPlacementScheme": "n/a"
+    "NIRSPlacementScheme": "10-20"
     ```
 
--   If an EasyCap was used but with custom positions,
-    as may be done by cutting custom holes in the cap,
-    was used:
+-   If an Artinis Medical Systems cap with custom positions,
+as may be done by cutting custom holes in the cap,
+was used:
     ```JSON
-    "CapManufacturer": "EasyCap",
-    "CapManufacturersModelName": "custom",
-    "NIRSPlacementScheme": "n/a"
+    "CapManufacturer": "Artinis Medical Systems",
+    "CapManufacturersModelName": "headcap with print, size L, it was modified by adding holes for the optodes according to the NIRSPlacementScheme and optode_layout.pdf",
+    "NIRSPlacementScheme": "see optode_layout.pdf: 2 groups over the left and right dlPFC, 2 groups over the left and right PPC, 1 group over the left M1 and PMC"
```

Comment:
if a PDF file is recommended in this example, I think it should be mentioned in the text that this file needs to be listed in .bidsignore

Reply:
Or `sourcedata/`?

Reply:
Alternative graphical depictions of the cap could be used, for example a photo, or a bitmap drawing, or a word document. I think that those documents themselves should not be part of the spec (and hence must be listed in the bidsignore for the validator)